pkg/apis/kops/cluster.go

```diff
@@ -273,28 +265,36 @@ type EtcdClusterSpec struct {
 
 type EtcdMemberSpec struct {
 	// Name is the name of the member within the etcd cluster
-	Name string  `json:"name,omitempty"`
-	Zone *string `json:"zone,omitempty"`
+	Name          string  `json:"name,omitempty"`
+	InstanceGroup *string `json:"instanceGroup,omitempty"`
 
 	VolumeType      *string `json:"volumeType,omitempty"`
 	VolumeSize      *int    `json:"volumeSize,omitempty"`
 	KmsKeyId        *string `json:"kmsKeyId,omitempty"`
 	EncryptedVolume *bool   `json:"encryptedVolume,omitempty"`
 }
 
-type ClusterZoneSpec struct {
-	Name string `json:"name,omitempty"`
+// SubnetType string describes subnet types (public, private, utility)
+type SubnetType string
 
-	// For Private network topologies we need to have 2
-	// CIDR blocks.
-	// 1 - Utility (Public) Subnets
-	// 2 - Operating (Private) Subnets
+const (
+	SubnetTypePublic  SubnetType = "Public"
+	SubnetTypePrivate SubnetType = "Private"
+	SubnetTypeUtility SubnetType = "Utility"
+)
+
+type ClusterSubnetSpec struct {
+	// TODO: Rename
```

Comment:
Yes, I probably should do this - I changed it from Name -> SubnetName to force compiler errors, because previous Name was actually Zone.  This rename will not be visible outside the code though.  But I'll do it!